backend/app/utils.py

```diff
@@ -104,3 +107,45 @@ def create_parent_child_edges(
             "districtr_map_uuid": districtr_map_uuid,
         },
     )
+
+
+def transform_bounding_box(bbox: list, source_epsg: str, target_epsg: str) -> list:
```

Comment:
H: I would much rather doing this in postgis. This logic looks quite complex and depends on an additional package, `osgeo`, which [seems largely to be unmaintained](https://pypi.org/project/osgeo/) seeing as it's on v0.0.1 since Jun '21. If we do do this in python I'd prefer to depend on packages like `fiona` and `shapely` since they're actively maintained and are likely to be added to our environment when we bring in MGGG packages to do things like contiguity.